wagtailio/static/sass/components/_cta.scss

```diff
@@ -0,0 +1,46 @@
+.cta {
+    $root: &;
+    padding: 25px 30px;
+    text-decoration: none;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border: 1px solid $color--off-black;
+    border-radius: 10px;
+
+    > svg {
+        transition: transform 0.3s;
+        margin: 0 15px 0 auto;
```

Comment:
Ben may be better placed to comment on this in the design review but I wonder if the `15px` could be removed on mobile to save space

Reply:
Suggestion: add `flex-shrink: 0` to the svg to prevent it being squashed when long titles are used

Reply:
Sure thing